### PR TITLE
✨ [Fluent UI Icons] -> [Icon Fonts]

### PR DESCRIPTION
**Pull request by max-programming**
# Resource Name - Fluent UI Icons

Fluent UI icons is an icon font provided by Microsoft.

Link - Fluent UI icons: https://developer.microsoft.com/en-us/fluentui?fabricVer=7#/get-started/web

#### Checklist:

- [x] I have performed a self-review of submitted resource and its follows the guidelines of the project.

### CODE REVIEW

**Comment by max-programming** [comment]
## Please don't merge now
Fluent UI icon's documentation is not that user friendly, its a bit cumbersome to set up. So I have a [blog post](https://blog.usman-s.me/how-to-use-microsoft-fluent-ui-icons-on-your-website) on how you can use Fluent UI icons in your project. Should I also add the blog post link with the docs link?

**Comment by max-programming** [comment]
image

☝️  This is how it looks like

**Comment by max-programming** [comment]
Let me know if any changes are required

File: readme.md [hunk 1]
```diff
@@ -279,6 +279,7 @@
 | [Font Awesome](https://fontawesome.com/)| The web's most popular icon set and toolkit  |
 | [Line Awesome](https://icons8.com/line-awesome)| Swap in replacement of Font Awesome with modern line icons |
 | [Material Icons](https://material.io/resources/icons/)| Material design icon library |
+| [Fluent UI Icons](https://developer.microsoft.com/en-us/fluentui#/styles/web/icons)| Fluent UI icons by Microsoft. [How to set it up?](https://blog.usman-s.me/how-to-use-microsoft-fluent-ui-icons-on-your-website) |
```
Comment (Alaev) [inline readme.md:282]: Please move it to react section as it's react UI.

Also please remove the how to setup link.

Thanks

**Review by Alaev — CHANGES_REQUESTED** [review]

**Comment by Alaev** [comment]
Closing as you can find the resource under `[Design Systems & Style Guides]` section.

**Comment by max-programming** [comment]
> Closing as you can find the resource under `[Design Systems & Style Guides]` section.

Yes I know about that one. It's not about icons whilst I added the Fluent UI icons in the font icons section

File: readme.md (hunk 1 above)
Comment (max-programming) [reply readme.md:282]: Not specifically React because it can be used in static sites with a CDN and in React with it's package

**Comment by Alaev** [comment]
Let's try to keep the list out of duplicated resources, Overall, they are under the same name, and most of the time design systems will have their own icons in it, you just need to search.

**Comment by max-programming** [comment]
Okay! No problem

**Pull request closed without merging**